src/components/RadioButtons/RadioButtons.js

```diff
@@ -98,64 +64,96 @@ RadioButton.propTypes = {
   onFocus: PropTypes.func,
 }
 
-/**
- * RadioButtonGroup returns a stack of the above `<RadioButton>`'s.
- *
- * It is mostly form-library-agnostic, to lessen our dependency on redux-form,
- * but there are a few aspects worth mentioning.
- *
- * If this is used as the component for a Field in the shared/fields folder,
- * (it will), it can expect to receive a `value` prop (via redux-form). We can
- * then use this value to determine which radio button should receive the
- * `checked` prop. (See similar ideas on redux-form #124: https://git.io/fj9Tb)
- *
- * If we change form libraries/approaches, this component might need tweaks.
- *
- * @param {String}     name      The name of the field, e.g. 'faveColor'
- * @param {Array}      options   Props passed to the individual radios
- * @param {String}     value     The currently selected option.value
- * @param {Boolean}    disabled  Whether this radio group is disabled
- * @param {Boolean}    required  Whether this radio group is required
- * @param {...Object}  rest      Mostly redux-form Field input/meta props
- */
 export function RadioButtonGroup({
-  name,
   options,
-  value,
+  onChange,
+  formChangeHandler,
+  name = `radio-button-group-${uuidv4()}`,
+  initialValue = undefined,
+  currentValue,
+  currentError,
+  formTouched,
   disabled,
+  validator,
   required,
-  ...rest // includes redux-form props, e.g. input.onChange
+  ...rest
 }) {
-  // Passing `disabled` at the group/field level disables all options. You can
-  // also disable an individual option/radio button via `option.disabled`.
-  // Not supported yet: autoComplete, tabIndex (not obviously necessary)
+  let initialSelected
+  if (currentValue || typeof currentValue === 'boolean') {
+    initialSelected = currentValue
+  } else if (initialValue || typeof initialValue === 'boolean') {
+    initialSelected = initialValue
+  }
+
+  const [selectedValue, setSelectedValue] = useState(initialSelected)
+  const [isAnswered, setIsAnswered] = useState(false)
+  const resolvedValidator = validator ? validator : () => ''
+  const [getError, setError, , validate] = useErrorMessage(resolvedValidator)
+
+  useEffect(() => {
+    const isSelectedValue = typeof selectedValue !== 'undefined'
+    if (onChange && isSelectedValue) {
+      onChange({ value: selectedValue, isAnswered })
+    }
+    if (formChangeHandler && isSelectedValue) {
+      // Ensure all validators get called
+      let errorMessage = validate(selectedValue)
+      errorMessage = errorMessage.length ? errorMessage : ''
+      setError(errorMessage)
```

Comment:
Errors are only handled and display if this is inside of a form and has the `formChangeHandler` supplied. So if you supply a validator to a standalone radiobuttongroup it can't show an error.

Reply:
Fixed:

![image](https://user-images.githubusercontent.com/142403/68330116-829af600-0087-11ea-83e1-082dad315526.png)

Reply:
>It's totally un-tabbable. Tabbing just skips over the field completely. 
Navigation is fixed and per the spec https://www.w3.org/TR/wai-aria-practices/examples/radio/radio-1/radio-1.html and if you tab in and then use arrow keys, space/enter to select etc.